[#119] - Preloading the template chunk for the certificate file

sumitnitsurat opened the pull request:
- added the preloading of templates before the verification starts.

rjchow commented:
@sumitnitsurat please review and write tests when you get back 😜

rjchow commented on src/sagas/templates.test.js:
```diff
@@ -0,0 +1,36 @@
+import { get } from "lodash";
+import { certificateData } from "@govtechsg/open-certificate";
+import templates from "../components/CertificateTemplates";
+import json from "../../tests_fixture/NP_Certs_FT_BMS.json";
```
i think just use a minimal cert inline instead of using a full cert

sumitnitsurat replied:
We have any function which returns the minimal cert or you want me to remove the data from this cert itself ?

rjchow replied:
you can use this: https://github.com/OpenCerts/opencerts-website/blob/master/src/sagas/testutils.js

rjchow committed:
[#119] - Preloading the template chunk for the certificate file

- added the preloading of templates before the verification starts.

[#119] - Fixed jest error for static files.

- added moduleNameMapper configuration in jest config file to mock the images, codecs or styling files.

[#119] - linting fix

[#119] - Preloading the template chunk for the certificate file

- added the preloading of templates before the verification starts.

- build fix

- remove pacage-lock.json and re-run npm install

- code restructuring

moved templates into its own saga and reducer


Revert "Sync with master"

This reverts commit 5009070c8ae27d333d1ef35a3c1b2c57565f3c55, reversing
changes made to 4b5b263f719ac75f784446eb90fb3633b1dd1a3d.

Revert "Revert "Sync with master""

This reverts commit 02e0c7a7cda75cee8bc389d8fc8006fac441f02e.

Lint and merge fix

- Test case for pre fetch template

Update package-lock.json

-  Minimal cert added

- added a function to add data field in minimal cert
